packages/terra-form-input/tests/wdio/form-input-spec.js

```diff
@@ -142,5 +144,24 @@ describe('Form-Input', () => {
         },
       });
     });
+
+    describe('InputField', () => {
```

Comment:
Is this test a duplicate of the valid and invalid test steps above?

Reply:
Since we added the toggle button to the field routes, I think we should take screenshots of both valid and invalid InputFields.